tests/tools/kopiaclient/kopiaclient.go

```diff
@@ -73,6 +75,43 @@ func (kc *KopiaClient) CreateOrConnectRepo(ctx context.Context, repoDir, bucketN
 	return errors.Wrap(err, "unable to open repository")
 }
 
+// ConnectRepo connects to an existing one if possible.
+func (kc *KopiaClient) ConnectRepo(ctx context.Context, repoDir, bucketName string) error {
+	st, err := kc.getStorage(ctx, repoDir, bucketName)
+	if err != nil {
+		return err
+	}
+
+	if iErr := repo.Connect(ctx, kc.configPath, st, kc.pw, &repo.ConnectOptions{}); iErr != nil {
+		return errors.Wrap(iErr, "error connecting to repository")
+	}
+
+	return errors.Wrap(err, "unable to open repository")
+}
+
+// SetCacheLimits connects to an existing one if possible.
+func (kc *KopiaClient) SetCacheLimits(ctx context.Context, repoDir, bucketName string, cacheOpts *content.CachingOptions) error {
+	err := kc.ConnectRepo(context.Background(), repoDir, bucketName)
+	if err != nil {
+		return err
+	}
+
+	err = repo.SetCachingOptions(ctx, kc.configPath, cacheOpts)
+	if err != nil {
+		return err
+	}
+
+	cacheOptsObtained, err := repo.GetCachingOptions(ctx, kc.configPath)
+	if err != nil {
+		return err
+	}
+
+	log.Printf("content cache size" + strconv.FormatInt(cacheOptsObtained.ContentCacheSizeLimitBytes, 10))
+	log.Printf("metadata cache size " + strconv.FormatInt(cacheOptsObtained.MetadataCacheSizeLimitBytes, 10))
+
+	return errors.Wrap(err, "unable to set the cache repository")
```

Comment:
It doesn't look like this `Wrap` does anything, since all non-nil errors are returned earlier. We can probably simplify to:
```suggestion
	return nil
```

Reply:
done